packages/dotcom-shared/src/tla-schema/TldrawAppUser.ts

```diff
@@ -6,6 +6,7 @@ import {
 	createRecordType,
 } from '@tldraw/store'
 import { T } from '@tldraw/validate'
+import { TLUserPreferences, getFreshUserPreferences, userTypeValidator } from 'tldraw'
```

Comment:
@ds300 this is actually causing problems with the bundle-size.
i tried changing it to import from `@tldraw/editor`, that helps but it still increases the bundle size significantly. maybe we move `TLUserPreferences` to schema?

Reply:
nvm, just seeing your other comment which invalidates this q